enclave/server.ts

```diff
@@ -51,11 +46,64 @@ const nStates = new ethers.Contract(
     signer
 );
 
+type ClaimedMove = {
+    uFrom: Tile;
+    uTo: Tile;
+};
+
 /*
  * Enclave's internal belief on game state stored in Board object.
  */
 let b: Board;
 
+/*
+ * Bijection between player's public keys and their socket IDs.
+ */
+let idToPubKey = new Map<string, string>();
```

Comment:
discussed this offline. cleaning up bijection as clients break connections + checking security if we authenticate by socket ID